<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastroVeiculos</class>
 <widget class="QDialog" name="cadastroVeiculos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela de Cadastro de Veiculos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_placa">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Placa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_modelo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Modelo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_marca">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Marca</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ano">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ano</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_quantidadeEixos">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de eixos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_criarVeiculo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSair">
    <property name="title">
     <string>Sair</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSair"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
